.swift-version

```diff
@@ -1 +1 @@
-5.2.4
+5.5.1
```

Comment:
Should this change be part of the PR? or should I somehow only do it locally? (It was required for tests to pass)

Reply:
hm I'm not sure if we want to bump the version here, @kmahar or @patrickfreed could probably give a better answer

Reply:
yeah I think it's fine to update this now

Reply:
Great, thanks